client/trino-jdbc/src/test/java/io/trino/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -411,15 +407,12 @@ public void testTimeWithTimeZone()
                 assertEquals(rs.getObject(column), Time.valueOf(LocalTime.of(1, 39, 7))); // TODO this should represent TIME '09:39:07 +01:00'
                 assertThatThrownBy(() -> rs.getDate(column))
                         .isInstanceOf(SQLException.class)
-                        .hasMessage(serverSupportsVariablePrecisionTimeWithTimeZone()
-                                ? "Expected value to be a date but is: 09:39:07+01:00"
-                                : "Expected value to be a date but is: 09:39:07.000 +01:00");
+                        .hasMessage("Expected value to be a date but is: 09:39:07+01:00");
                 assertEquals(rs.getTime(column), Time.valueOf(LocalTime.of(1, 39, 7))); // TODO this should fail, or represent TIME '09:39:07'
+                // TODO (https://github.com/trinodb/trino/issues/5317) placement of precision parameter
```

Comment:
Since the assertions are fluent and thus multi-step, this comment should be retained back on (or before) the line it pertains to.